### PR TITLE
version 1.1.5 for PhpStorm 2024.1 

### PR DESCRIPTION
**Pull request by Hidanio**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -5,25 +5,25 @@ pluginName = modulite
...
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
 # PHP Plugin - https://plugins.jetbrains.com/plugin/6610-php/versions
 # YAML Plugin - https://plugins.jetbrains.com/plugin/13126-yaml/versions
-platformPlugins = com.jetbrains.php:233.11799.232, org.jetbrains.plugins.yaml:233.11799.165
+platformPlugins = com.jetbrains.php:241.14494.237, org.jetbrains.plugins.yaml:241.14494.150
 
 # Gradle Releases -> https://github.com/gradle/gradle/releases
-gradleVersion = 8.5
+gradleVersion = 8.7
```
Comment (Danil42Russia) [inline gradle.properties]: `8.6`

File: gradle.properties
Comment (Danil42Russia) [inline gradle.properties:1]: Remove `systemProp.org.gradle.unsafe.kotlin.assignment`

**Review by Danil42Russia — DISMISSED** [review]

File: gradle.properties (hunk 1 above)
Comment (Hidanio) [reply gradle.properties]: Done

File: gradle.properties
Comment (Hidanio) [reply gradle.properties:1]: Changed to false. Let it be clearly turned off

Comment (Danil42Russia) [reply gradle.properties:1]: No, you don't have to do that.
`systemProp.org.gradle.unsafe.kotlin.assignment` is now enabled by default as it is. Remove this block.
We're trying to be consistent with [JetBrains/intellij-platform-plugin-template](https://github.com/JetBrains/intellij-platform-plugin-template)

File: .github/workflows/build.yml
Comment (Danil42Russia) [inline .github/workflows/build.yml:1]: And where is the `concurrency` block?

File: build.gradle.kts [hunk 2]
```diff
@@ -19,11 +19,7 @@ repositories {
 
 // Set the JVM language level used to build the project. Use Java 11 for 2020.3+, and Java 17 for 2022.2+.
```
Comment (Danil42Russia) [inline build.gradle.kts]: The comment also needs to be updated.

This is done to transfer changes from [JetBrains/intellij-platform-plugin-template](https://github.com/JetBrains/intellij-platform-plugin-template) one-to-one.

File: gradle/wrapper/gradle-wrapper.jar
Comment (Danil42Russia) [inline gradle/wrapper/gradle-wrapper.jar:1]: Changed the `gradle` version, but did not update the `wrapper` (from `8.7` to `8.6`)

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+# 1.1.5 - 09.04.2024
+
+- Adapt code for PhpStorm 2024.1
```
Comment (Danil42Russia) [inline CHANGELOG.md:7]: `.run/Run Tests.run.xml` did not change

File: gradle.properties
Comment (Hidanio) [reply gradle.properties:1]: Done

File: .github/workflows/build.yml
Comment (Hidanio) [reply .github/workflows/build.yml:1]: Fixed

File: build.gradle.kts (hunk 2 above)
Comment (Hidanio) [reply build.gradle.kts]: Done

File: gradle/wrapper/gradle-wrapper.jar
Comment (Hidanio) [reply gradle/wrapper/gradle-wrapper.jar:1]: Updated

File: CHANGELOG.md (hunk 3 above)
Comment (Hidanio) [reply CHANGELOG.md:7]: Done

**Review by Danil42Russia — APPROVED** [review]

**Pull request merged**